fedimint-core/Cargo.toml

```diff
@@ -41,6 +41,7 @@ hbbft = { git = "https://github.com/jkitman/hbbft", branch = "upgrade-threshold-
 rand = "0.8.5"
 miniscript = { version = "7.0.0", git = "https://github.com/rust-bitcoin/rust-miniscript/", rev = "2f1535e470c75fad85dbad8633986aae36a89a92", features = [ "compiler", "serde" ] }
 secp256k1-zkp = { version = "0.7.0", features = [ "use-serde", "bitcoin_hashes", "global-context" ] }
+serde_with = "2.2.0"
```

Comment:
that would be the introduction of a new dependency; though, I guess that'd be fine?

Reply:
I can implement it manually, if we don't want to be have this dependency.